src/commands/cloud-run/README.md

```diff
@@ -0,0 +1,53 @@
+## Troubleshooting Cloud Run Instrumentation
+
+To troubleshoot issues you encounter with Datadog monitoring on your Cloud Run services, run the `datadog-ci cloud-run flare` command in the root of your project directory. This command collects important data about the Cloud Run service, such as environment variables and configuration information. These files will be submitted to Datadog support via a ticket matching the provided Zendesk case ID.
+
+## Configuration
+
+### GCP Credentials
+
+You must have valid [GCP credentials][1] configured with access to the Lambda and CloudWatch services where you are running any `datadog-ci lambda` command. You can configure credentials by running `gcloud auth application-default login` and following the prompts in your browser. 
+
+### Environment variables
+
+Expose these environment variables in the environment where you are running `datadog-ci cloud-run flare`:
+
+| Environment Variable | Description                                                                                                                                                                                                                                      | Example                               |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
+| `DATADOG_API_KEY`    | Datadog API Key. Used to attach the flare files to your Zendesk ticket. For more information about getting a Datadog API key, see the [API key documentation][2].                                                                                | `export DATADOG_API_KEY=<API_KEY>`    |
```

Comment:
Should we change this to `DD_API_KEY` and `DD_SITE` since we already support those?